pkg/common/utils.go

```diff
@@ -284,6 +284,17 @@ func ConvertMiStringToInt64(str string) (int64, error) {
 	return volumehelpers.RoundUpToMiB(quantity)
 }
 
+// ConvertStringToBool converts a true/false string into a bool, or error.
+func ConvertStringToBool(str string) (bool, error) {
```

Comment:
I feel like there should be a standard k8s conversion library to use, but I couldn't find it.

Reply:
k8s api convention is to avoid bools as much as possible, because in the future if you want to add a 3rd value it gets hard: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types

Reply:
Yes, however the gce api this maps to is a bool: https://cloud.google.com/compute/docs/reference/rest/v1/instances/attachDisk

I think at this point keeping this mapping direct is best, we don't know what abstraction layer we want to have between this driver and the api yet.

Reply:
You can still validate the values. It does shift the meaning of the field to express a higher level intent and opens it up to more options in the future

Reply:
Yeah, fair enough. Anyway we don't need to decide that now. The concern I have is that if change things we may not use some of these alpha values but we don't want to reject old storage classes that still have the parameters. Maybe there's some way to do a warning.